src/products/controller/productCategories.controller.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import ProductCategoriesMapper from '../mapper/productCategory.mapper';
+import { ProductCategoriesService } from '../service/productCategories.service';
+import ProductCategory from '../domain/productCategory.domain';
+import ProductCategoryDTO from '../dto/productCatergory.dto';
+import { ProductsService } from '../service/products.service';
+import { ApiResponse } from '@nestjs/swagger';
+
+@Controller('productCategories')
```

Comment:
Check this: [REST API Naming Conventions and Best Practices](https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5)

Meaning: `product-categories` is preferred.

Reply:
done